sample-plugin/pom.xml

```diff
@@ -515,14 +515,14 @@
             <id>2.346.x</id>
             <properties>
                 <bom>2.346.x</bom>
-                <jenkins.version>2.346</jenkins.version>
+                <jenkins.version>2.346.1</jenkins.version>
             </properties>
         </profile>
         <profile>
             <id>2.332.x</id>
             <properties>
                 <bom>2.332.x</bom>
-                <jenkins.version>2.332.1</jenkins.version>
+                <jenkins.version>2.332.4</jenkins.version>
```

Comment:
Somehow the current doc shows 2.332.3, FYI @daniel-beck (context: https://github.com/jenkins-infra/jenkins.io/pull/4876)

Reply:
https://github.com/jglick/jenkins.io/blob/aed4f7fd2d70b6325d447d00588d9d86c0acd2a8/content/_layouts/corebaseline.html.haml#L13 is just https://updates.jenkins.io/tiers.json via https://github.com/jenkins-infra/jenkins.io/blob/64ed7120a01fde6d35946423e35da51e4a95fca3/scripts/fetch-external-resources#L54-L55 and it lists versions actually used as dependency only (so we can generate matching update sites).

I guess I was lazy when I set this up and didn't want to deal with Artifactory API.

By now there's half a dozen plugins that depends on 2.332.4, so the site is fixed.

Reply:
⇒ https://github.com/jenkins-infra/jenkins.io/pull/5320